cl/audio/tests.py

```diff
@@ -66,7 +66,7 @@ def test_do_search_podcasts_have_content(self) -> None:
         response = self.client.get(
             reverse("search_podcast", args=["search"]),
             {
-                "q": f"court:{self.audio.docket.court}",
+                "q": f"court_id:{self.audio.docket.court.pk}",
```

Comment:
This tweak surprised me a bit, but it looks like `court` wasn't a documented filter previously, so we're OK. One thing though, I noticed [in the documentation](https://www.courtlistener.com/help/search-operators/#fielded-queries-fieldname-term) that judges can still be filtered by `court` instead of `court_id`. Is that right and should we fix it? 

![image](https://github.com/freelawproject/courtlistener/assets/236970/5e02efe2-f0ac-4276-9cd6-f8b2a2af569b)

Reply:
Oh yeah, I changed this test because previously I removed all the fields in `audio_text.txt` and the tests regarding podcast in this PR were working on the Solr version. So this test was failing due to the court was not being indexed anymore, so I changed it to `court_id` which is the documented filter. 

I've updated this PR with it's base branch where Podcast were moved to ES. So this is not an issue anymore, however I still think the `court_id` filter is the right one in this test, right?

Then I restored content in `audio_text.txt` (which is now used only for Solr) since I realized will continue indexing OA documents in Solr for some time before we switch to ES completely, then we can remove the whole file.

About Judges, yeah we use `court_exact` for the courts filter, so maybe we should change it to `court_id` and adding it to documentation? `court` holds the court full name.

Reply:
> I still think the court_id filter is the right one in this test, right?

Seems right to me.

> we should change it to court_id and adding it to documentation?

I agree.